frontend/src/pages/schedules/index.tsx

```
@@ -62,11 +67,32 @@ function SchedulesIndexPage() {
+	const [viewMode, setViewMode] = useState<"calendar" | "table">("calendar");
 	const [showCreateDialog, setShowCreateDialog] = useState(false);
 	const [showEditDialog, setShowEditDialog] = useState(false);
 	const [selectedAgentId, setSelectedAgentId] = useState<string>("");
 	const [editingSchedule, setEditingSchedule] = useState<Schedule | null>(null);
 
+	const { executions } = useScheduleExecutions();
+
+	// Map executions to calendar events
+	const schedulesMap = useMemo(() => {
+		const map = new Map<string, Schedule>();
+		schedules.forEach((s) => map.set(s.id, s));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)